### DISCUSSION TITLE
SMV issue - Suddent interupt when loading Soot, HRRPUV or Temperature

### DISCUSSION THREAD

**Discussion by Heisenberg-1998** [firemodels/fds#13259] [Q&A]
Hi everyone,

I ran a validated model for Kirby Test on computer 1. After finishing the calculation, I copied - patsted all output files to computer 2. But when I open SmokeView and load the soot, HRRPUV or temperature, the SmokeView will suddenly interupt. However I this issue didn't come out on computer 1. I have double checked I copied all the output files on computer 1. 

Dose anyone has the similar issue and know why it happend? :)

**Comment by Heisenberg-1998** [comment]
![image](https://github.com/user-attachments/assets/6c89635e-37b5-43b5-aa76-2692c080ebdc)
![image](https://github.com/user-attachments/assets/5707062d-b106-4301-a9d5-c9ff080eff5f)

Here are two loading processes for Soot, on computer 2. The loading process seems to complete, but the  SMV programme window suddenly disappears after that.

**Comment by gforney** [comment]
What version of smokeview are you using? ( smokeview -v).  Uplooad your
input file and ill take a look at it

**Reply by Heisenberg-1998** [reply]
Hi gorney,

The smokeview version is 6.9.1.0. The 'input file' you are refering to, it is the FDS input file? 

[BST_FRS_1.txt](https://github.com/user-attachments/files/16465033/BST_FRS_1.txt)

Revision         : SMV-6.9.1-0-gd0b04d0-release
Revision Date    : Tue Apr 9 09:57:29 2024 -0400
Compilation Date : Apr  9 2024 13:42:40
Compiler         : Intel  2023.1.0
Checksum(SHA1)   : 8006730a5f722ec45a7f03601c679a8d2c1aa312
Platform         : WIN64  (Intel C/C++)
